### PR TITLE
Document return values for command module

### PR DESCRIPTION
**Pull request by prasadkatti**
##### SUMMARY
Document return values for command module

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
`lib/ansible/modules/commands/command.py`

##### ANSIBLE VERSION

```
ansible 2.4.0.0
  config file = /etc/ansible/ansible.cfg
  python version = 2.7.6 (default, Oct 26 2016, 20:30:19) [GCC 4.8.4]
```

### CODE REVIEW

File: lib/ansible/modules/commands/command.py [hunk 1]
```diff
@@ -81,6 +81,31 @@
...
+  returned: always
+  type: list
+  sample:
+  - echo
+  - hello
+delta:
+  description: cmd end time - cmd start time
+  returned: always
+  type: string
+  sample: '0:00:00.001529'
```
Comment (dagwieers) [inline lib/ansible/modules/commands/command.py]: Please don't add quotes when they're not necessary.
People may have te impression quoting is required here, but it's not.

File: lib/ansible/modules/commands/command.py [hunk 2]
```diff
@@ -81,6 +81,31 @@
...
+end:
+  description: cmd end time
+  returned: always
+  type: string
+  sample: '2017-09-29 22:03:48.084657'
```
Comment (dagwieers) [inline lib/ansible/modules/commands/command.py:101]: Same here.

File: lib/ansible/modules/commands/command.py [hunk 3]
```diff
@@ -81,6 +81,31 @@
...
+start:
+  description: cmd start time
+  returned: always
+  type: string
+  sample: '2017-09-29 22:03:48.083128'
```
Comment (dagwieers) [inline lib/ansible/modules/commands/command.py:106]: Same here.

**Review by dagwieers — CHANGES_REQUESTED** [review]
Looking good.

**Review by dagwieers — APPROVED** [review]

**Comment by sivel** [comment]
The values that look like dates *do* need quoted, otherwise YAML parsing will turn them into datetime objects, as evidenced by the CI failures

**Comment by prasadkatti** [comment]
@dagwieers Thanks for the review.
@sivel Thanks. I have added the quotes back to the `start` and `end` sample values.

**Comment by dagwieers** [comment]
Alright, learned something new today.

**Pull request merged**